apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,27 +1,17 @@
-<div
-  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
-  [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+<ng-content [card-image] />
 
-  <section>
-    <app-list-item
-      *ngFor="let item of list"
-      [name]="item.firstname"
-      [id]="item.id"
-      [type]="type">
-    </app-list-item>
-  </section>
+<section *ngIf="list">
+  @for (item of list; track $index) {
+    @if (row) {
```

Comment:
This is a good point on testing this, but you should prefer to display a default template, because the next developer or the future you can have a hard time debugger this.